pkg/asset/manifests/network-operator.go

```diff
@@ -1,72 +1,117 @@
 package manifests
 
 import (
+	"os"
+	"path/filepath"
+
 	"github.com/ghodss/yaml"
 	"github.com/pkg/errors"
 
 	"github.com/openshift/installer/pkg/asset"
 	"github.com/openshift/installer/pkg/asset/installconfig"
 
-	tectonicnetwork "github.com/coreos/tectonic-config/config/tectonic-network"
+	netopv1 "github.com/openshift/cluster-network-operator/pkg/apis/networkoperator/v1"
 	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
 )
 
+var (
+	noCrdFilename = filepath.Join(manifestDir, "cluster-network-01-crd.yml")
+	noCfgFilename = filepath.Join(manifestDir, "cluster-network-02-config.yml")
+)
+
 const (
-	defaultMTU         = "1450"
-	noCfgFilename      = "network-operator-config.yml"
-	noManifestFilename = "network-operator-manifests.yml"
+
+	// we need to manually create our CRD first, so we can create the
+	// configuration instance of it
+	netConfigCRD = `
+---
+apiVersion: apiextensions.k8s.io/v1beta1
+kind: CustomResourceDefinition
+metadata:
+  name: networkconfigs.networkoperator.openshift.io
+spec:
+  group: networkoperator.openshift.io
+  names:
+    kind: NetworkConfig
+    listKind: NetworkConfigList
+    plural: networkconfigs
+    singular: networkconfig
+  scope: Cluster
+  versions:
+    - name: v1
+      served: true
+      storage: true
+`
 )
 
-// networkOperator generates the network-operator-*.yml files
+// Network generates the cluster-network-*.yml files
 type networkOperator struct {
-	config   *tectonicnetwork.OperatorConfig
+	config   *netopv1.NetworkConfig
 	FileList []*asset.File
 }
 
 var _ asset.WritableAsset = (*networkOperator)(nil)
 
 // Name returns a human friendly name for the operator
 func (no *networkOperator) Name() string {
-	return "Network Operator"
+	return "Network Config"
 }
 
-// Dependencies returns all of the dependencies directly needed by an
-// networkOperator asset.
+// Dependencies returns all of the dependencies directly needed to generate
+// network configuration
 func (no *networkOperator) Dependencies() []asset.Asset {
 	return []asset.Asset{
 		&installconfig.InstallConfig{},
 	}
 }
 
-// Generate generates the network-operator-config.yml and network-operator-manifest.yml files
+// Generate generates the network operator config and its CRD
```

Comment:
nit: This was true of the previous comment as well, but can we get a trailing period here?  ([docs][1])

[1]: https://github.com/golang/go/wiki/CodeReviewComments#comment-sentences